<!DOCTYPE html>
<html><head><title>GeneralError</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">12 references to GeneralError</div><div class="rA">Azure.Messaging.EventHubs (4)</div><div class="rG" id="Azure.Messaging.EventHubs"><div class="rF"><div class="rN">EventHubsException.cs (3)</div>
<a href="../EventHubsException.cs.html#68"><b>68</b>string eventHubName) : this(isTransient, eventHubName, null, FailureReason.<i>GeneralError</i>, null)</a>
<a href="../EventHubsException.cs.html#96"><b>96</b>string message) : this(isTransient, eventHubName, message, FailureReason.<i>GeneralError</i>, null)</a>
<a href="../EventHubsException.cs.html#128"><b>128</b>Exception innerException) : this(isTransient, eventHubName, message, FailureReason.<i>GeneralError</i>, innerException)</a>
</div>
<div class="rF"><div class="rN">Primitives\EventProcessor.cs (1)</div>
<a href="../Primitives/EventProcessor.cs.html#1520"><b>1520</b>var slowException = new EventHubsException(true, EventHubName, message, EventHubsException.FailureReason.<i>GeneralError</i>);</a>
</div>
</div>
<div class="rA">Azure.Messaging.EventHubs.Shared.Tests (1)</div><div class="rG" id="Azure.Messaging.EventHubs.Shared.Tests"><div class="rF"><div class="rN">RetryPolicies\BasicRetryPolicyTests.cs (1)</div>
<a href="../../Azure.Messaging.EventHubs.Shared.Tests/RetryPolicies/BasicRetryPolicyTests.cs.html#259"><b>259</b>var exception = new EventHubsException(true, &quot;dummy&quot;, EventHubsException.FailureReason.<i>GeneralError</i>);</a>
</div>
</div>
<div class="rA">Azure.Messaging.EventHubs.Tests (7)</div><div class="rG" id="Azure.Messaging.EventHubs.Tests"><div class="rF"><div class="rN">Amqp\AmqpConsumerTests.cs (1)</div>
<a href="../../Azure.Messaging.EventHubs.Tests/Amqp/AmqpConsumerTests.cs.html#50"><b>50</b>yield return new object[] { new EventHubsException(false, &quot;test&quot;, &quot;fake&quot;, EventHubsException.FailureReason.<i>GeneralError</i>) };</a>
</div>
<div class="rF"><div class="rN">Amqp\AmqpExceptionExtensionsTests.cs (1)</div>
<a href="../../Azure.Messaging.EventHubs.Tests/Amqp/AmqpExceptionExtensionsTests.cs.html#59"><b>59</b>yield return new[] { new EventHubsException(true, &quot;fake&quot;, &quot;Some message&quot;, EventHubsException.FailureReason.<i>GeneralError</i>) };</a>
</div>
<div class="rF"><div class="rN">Primitives\EventProcessorTests.MainProcessingLoop.cs (4)</div>
<a href="../../Azure.Messaging.EventHubs.Tests/Primitives/EventProcessorTests.MainProcessingLoop.cs.html#271"><b>271</b>var expectedException = new EventHubsException(false, &quot;eh&quot;, &quot;LB FAIL&quot;, EventHubsException.FailureReason.<i>GeneralError</i>, wrappedExcepton);</a>
<a href="../../Azure.Messaging.EventHubs.Tests/Primitives/EventProcessorTests.MainProcessingLoop.cs.html#353"><b>353</b>var expectedException = new EventHubsException(false, &quot;eh&quot;, &quot;LB FAIL&quot;, EventHubsException.FailureReason.<i>GeneralError</i>);</a>
<a href="../../Azure.Messaging.EventHubs.Tests/Primitives/EventProcessorTests.MainProcessingLoop.cs.html#428"><b>428</b>var expectedException = new EventHubsException(false, &quot;eh&quot;, &quot;LB FAIL&quot;, EventHubsException.FailureReason.<i>GeneralError</i>, wrappedExcepton);</a>
<a href="../../Azure.Messaging.EventHubs.Tests/Primitives/EventProcessorTests.MainProcessingLoop.cs.html#1778"><b>1778</b>&amp;&amp; (((EventHubsException)value).Reason == EventHubsException.FailureReason.<i>GeneralError</i>)</a>
</div>
<div class="rF"><div class="rN">Primitives\EventProcessorTests.PartitionProcessing.cs (1)</div>
<a href="../../Azure.Messaging.EventHubs.Tests/Primitives/EventProcessorTests.PartitionProcessing.cs.html#1223"><b>1223</b>var expectedException = new EventHubsException(true, &quot;frank&quot;, &quot;BOOM!&quot;, EventHubsException.FailureReason.<i>GeneralError</i>);</a>
</div>
</div>
</body></html>